### DISCUSSION TITLE
How to deploy Redash on Multi Node Cluster

### DISCUSSION THREAD

**Discussion by subratshasank** [getredash/redash#6765] [Q&A]
The redash deployment/architecture has many components

Postgres
redis
redash server
NGINX
Celery Workers
gunicorn
we would like to break them up into two  independent services, Postgres , redash server , celery executor on one node and Celery worker on another node

**Comment by subratshasank** [comment]
Waiting for someone to answer thsi from past 15 days :(

**Comment by ognjen-it** [comment]
Hi @subratshasank ,

You can use tolerations and annotations to do this, helm chart from contrib-helm-chart repo has these options in values.yaml.

**Reply by subratshasank** [reply]
We didnot wanted to go on to K8 but multi node cluster where adding worker nodes could give us horizontal scalling

**Reply by ognjen-it** [reply]
What is used for orchestrating? Docker swarm?
